### ISSUE TITLE
Important dates for the end of the course

### ISSUE THREAD

**Issue by mikeizbicki** [mikeizbicki/ucr-cs100#562]
1. HW4 is due this Thursday (11 Dec) at midnight.  I'm willing to grant extensions to anyone who wants them if it will result in a better final product. In order to get an extension, please reply to this post with the following: the date you plan to submit it and a one sentence description of what you will do with the extra time.  I expect that for many submissions I will have feedback that you can incorporate to improve your grade.  The earlier you submit, the more time you'll have to respond to this feedback.
2. All of your grades should be entered into the system by the end of this week.  This includes  extra credit and quiz results that haven't been entered yet.
3. You will have until Friday of next week (19 Dec) to submit extra credit for the course.  After that time your grade will be final.  Most of this extra credit can be submitted and graded via github.  If you want to meet in person to discuss something, then reply to this post with the times you're available to meet.

**Comment by asand017** [comment]
I would like a two day extension, so Saturday (13 Dec) at midnight. I want to use the extra two days to better wrap my head around the less obvious uses of "sed" (i.e. experimentation, demonstration to include in the markup) and to ensure the language in my project is as clear and succinct as possible.

**Comment by holidaylvr** [comment]
@bchen022 and I will like a two day extension to Saturday (Dec 13) at midnight as well. We would like these two extra days so that we can receive feedback from you and fix any technicalities in our project that remains since we want to be as accurate as possible with our use of technical terms.

**Comment by hray001** [comment]
I would also like a two day extension to Saturday (Dec 13) for hw4. I want to be able to visit the writing center for the assignment on Friday and deal with the clarity of my tutorial as I realize it is too wordy.

EDIT:
I am about finished with one of the extra credit readings and it says to meet with you to discuss the reading. Also for the vim-adventures game would proof for that be submitted via github or in person?
I am free anytime next week besides-
Tuesday 8:00-11:00A.M.
Thursday 12:00-3:00P.M.
Friday 8:00-11:00A.M.

**Comment by emullally** [comment]
I would like to request a two day extension to Saturday Dec 13th at midnight for hw4, as well. I would use the time to expand on some of my coding examples for the new C++11 features I am explaining so they can better demonstrate advantages/uses of the features discussed. Also, sections of my tutorial could be more succinct and I could use the time to improve my writing.

**Comment by jbarb011** [comment]
I would like to request a four day extension to Monday Dec 15th at midnight. My project is a series of short animated videos and I want to make sure that they are of high quality but it takes a lot of time and effort to animate and edit these. I know that this is more time than what most of the other students are asking for, so I would like to discuss if this is an appropriate amount of time in office hours later today.

**Comment by mikeizbicki** [comment]
@jbarb011 That amount of time is fine, but make sure to get me a sample video earlier so I can give you some feedback on if you're going in the right direction.

**Comment by khiguera** [comment]
I would like to request a two day extension as well to Saturday, Dec. 13 at midnight in order to create more code examples. Also, in order to integrate the feedback given and establish the organization you mentioned in your feedback.

**Comment by ogonz007** [comment]
I would like to request a extension until midnight Saturday Dec 13th. We would like to spend more time implementing a visual representation of the output of our neuron model as well as ensuring people outside of neuroscience can understand our psudocode examples.

**Comment by ahsu013** [comment]
I would like a Two day extension to Saturday, Dec. 13 at midnight. I would use this time to fix all my mistakes, add more content , and fix the format.

**Comment by ziyaemanet** [comment]
I would like to request a extension until midnight Wednesday Dec 17th. I have run into a dead end with my idea for hw4 so I began hw4 on IOCCC 2012 winner of most elementary use of C - hamano. This extra time would be used to more thoroughly explain the obfuscation methods used, create graphics of code, and to request feedback regarding the project.

**Comment by atosti** [comment]
I would like a four day extension until midnight on Monday, December 15th. I plan to add more applications of Tribool in combination with a container class to my boost tutorial, in order to provide more of an example-based documentation.

**Comment by thinkaliker** [comment]
@Liniarc and I would like an extension to Saturday, December 13 (midnight). We are going to move our regex program to it's own repository and add features.

**Comment by Daniel-Goerlich** [comment]
Kevin and I would like an extension until Monday, December 15th at midnight. We would like the extra time to get more feed back on the current state of our work on GDB. In addition, the extra time may allows to find more interesting facts and more useful commands in GDB to add for extra credit on the assignment.

**Comment by nmlapre** [comment]
I would like an extension to Sunday, December 14th at midnight. I will use that time to expand and touch up my tutorial, as well as generate a sample .bashrc file that specifically CS100 students may find useful.

**Comment by anmolsingh95** [comment]
I have sent en email to the XMonad mailing list for a review of my tutorial. i wish an extension until Saturday. I feel like their review might point out any more inaccuracies, and will help make the tutorial better.

**Comment by rtran012** [comment]
I would like an extension till Sunday, December 14 at midnight.  I will use that time to expand and provide more extensive examples.

**Comment by hgarc014** [comment]
Daniel and I would like an extension on our git-game. We are asking for the deadline to be Thursday December 18. We will use the extra time to develop a hard version of the git-game for more experienced git users.

**Comment by ifuka001** [comment]
@jesuscreyes and I would like an extension until Sunday. We will use the time to get the layout better and add more commands like you've suggested.

**Comment by jhollister** [comment]
I would like an extension until Sunday, midnight for my hw4 on tmux. I've run into a few issues with the overall structure of my guide and I think I will be able to make a lot of improvements with this time.

**Comment by mxu008** [comment]
I would like an extension until midnight of Sunday. I need some time to make some changes for my hw4 based on the comments from you. And I have two finals in Saturday so that I really need a extension.

**Comment by gbrlrz017** [comment]
I would like an extension to Tuesday night. I will use this time to provide more examples on how to come up with test cases, make some more rough draft submissions for feed back, and research the methods established programmers use.

**Comment by jmoon018** [comment]
I'd like an extension until Monday at midnight for both Pac-Vim and the unit-tester. There are still a few new features I need to implement and test for Pac-Vim, which I suspect will eat up some time. For the unit-tester, I just have to refactor; I think it may work for beyond HW#0, but it might require a few more adjustments.

**Comment by palvarezz** [comment]
I'd like an extension until Sunday so I could come up with better examples and make some more changes to my hw4.

**Comment by jacksonfan** [comment]
@rluk001 and I would like an extension until tomorrow at midnight so we can implement HTML data tables and format everything to look neat and professional. We'd also be polishing up the language and the structure of our writeup

**Comment by atan009** [comment]
I and hfanc001 would like an extension until wednesday so that we may add more gifs, fix formatting as well as clarify some areas. We will also add a section about how to install the distribution of our vimrc and make changes to it.

**Comment by jinsongwei** [comment]
I would like one day extension. I plan to meet writing center to review my tutorial before submitting on Friday

**Comment by Brady1026** [comment]
I'd like to request an extension until at least Sunday because what I have right now is insufficient.

**Comment by ChemicalWonders** [comment]
I would like an extension until Wednesday to write up a bit more on the Advanced Git tutorial.

**Comment by Dave864** [comment]
I have finished my first draft. What are your thoughts on it?
Here is the link to it:
[How to Use Strtok](https://github.com/Dave864/rshell/tree/master/src)

**Comment by Daniel-Goerlich** [comment]
If possible I woud like to meet Tuesday, December 16 at 5:00pm to discuss one of the extra credit readings and show that I have linux on my laptop. In addition, will the meetings be at the usual office hours location?

**Comment by jbarb011** [comment]
I have submitted my hw4 project "Syscalls 4 Dummies", however it is incomplete. I ran into some problems with Premiere and ended up getting a lot of my footage corrupted, leaving me having to redo a lot of the project. I will have the last video up by Wednesday night at the latest, whether or not you give me another extension. I'm sorry for the inconvenience.

**Comment by hray001** [comment]
Is there any time on Friday I could meet with you on Friday December 19 to discuss one of the extra credit readings? I would like to meet sometime around 1:00pm unless there is another time which is more convenient.

**Comment by ziyaemanet** [comment]
I really need until Friday. I ran into some issues with the recursive portion of the obfuscated code. I will submit a pull request later tonight for review of current progress. 

If this could happen I would appreciate it.

**Comment by mikeizbicki** [comment]
@ziyaemanet you have until Friday
